frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/__tests__/PipelineParameters.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import PipelineForm from '../../pipeline-form/PipelineForm';
+import PipelineParameters from '../PipelineParameters';
```

Comment:
This file is actually a "`PipelineForm` test"... we should just validate it shows a `Formik` component that somewhere beneath it shows `PipelineParameters` component; hopefully that doesn't couple to the Formik implementation and cause erroneous test breaks if we update `Formik`... I think it's okay, but we have to be careful not to assume anything about the `Formik` component structure and just know it needs to preset our rendered component at some point.

Reply:
You could probably also add a test to validate the `initialValues` prop in `Formik` but it shouldn't go any further than that.